src/tests/multihost/alltests/test_sssd_nss.py

```diff
@@ -0,0 +1,65 @@
+""" Automation of sssd nss suite
+
+:requirement: sssd_nss
```

Comment:
Currently there is no requirement `sssd_nss`. We can use `IDM-SSSD-REQ : Configuration and Service Management` here. Add the exact string to enable this new test to get linked by the importer

Reply:
changes done

Reply:
Done